app/src/main/java/com/example/coctailapp/MainActivity.kt

```diff
@@ -14,21 +14,21 @@ import androidx.core.splashscreen.SplashScreen.Companion.installSplashScreen
 import androidx.navigation.compose.NavHost
 import androidx.navigation.compose.composable
 import androidx.navigation.compose.rememberNavController
-import com.example.coctailapp.ui.screens.loginScreen
```

Comment:
Please rename the .kt files to upper case as well to reflect the composable name